source/game.ingameinterface.bmx

```diff
@@ -582,7 +583,7 @@ Type TInGameInterface
 
 				MenuToolTip.area.SetX(364)
 				MenuToolTip.SetTitle(getLocale("MENU"))
-				MenuToolTip.SetContent(getLocale("OPEN_MENU"))
+				MenuToolTip.SetContent("(Esc) " + getLocale("OPEN_MENU"))
```

Comment:
Hier bin ich mir nicht sicher, ob man die Tastenkuerzel nicht alle "uppercase" schreiben sollte.
`(ESC)` vs `(Esc)`
auch ist mein _persoenlicher_ Favorit die eckige Klammer:
`(ESC)` vs `(Esc)` vs `[ESC]` vs `[Esc]`

Gern koennten wir auch die Kuerzel einfaerben bzw. schraegstellen/fettschreiben/...

Reply:
Das kann ich aber gern in einem Folgecommit anpassen - falls moeglich.

Reply:
Uppercase und eckige Klammern OK. Die andere Hervorhebung ist vielleicht für das erste Wahrnehmen hilfreich, dauerhaft aber zu aufdinglich (finde ich).